server/storages/factory.go

```diff
@@ -68,8 +70,13 @@ func (ur *UsersRecognition) Validate() error {
 			return errors.New("anonymous_id_node is required")
 		}
 
-		if ur.UserIDNode == "" {
-			return errors.New("user_id_node is required")
+		if len(ur.IdentificationNodes) == 0 {
+			if ur.UserIDNode == "" {
+				return errors.New("identification_nodes is required")
+			} else {
+				logging.Warn("user_id_node is deprecated. Please use identification_nodes instead")
```

Comment:
Please add full path to config parameter:

```
logging.Warn("users_recognition.user_id_node is deprecated. Please use users_recognition.identification_nodes instead")
```

Reply:
Changed